src/App.jsx

```diff
@@ -13,11 +13,11 @@ export const tabs = [
 export const App = () => {
   const [selectedTab, setSelectedTab] = useState(tabs[0]);
 
-  const onTabSelected = (tab) => {
-    if (tab.id !== selectedTab.id) {
-      setSelectedTab(tab);
-    }
-  };
+  // const onTabSelected = (tab) => {
+  //   if (tab.id !== selectedTab.id) {
+  //     setSelectedTab(tab);
+  //   }
+  // };
```

Comment:
if you don't use comments or comments don't explain anything, it's better to remove them

Reply:
please delete this commented function =)